flask_celeryext/app.py

```diff
@@ -83,7 +83,6 @@ class RequestContextTask(Task):
     def __call__(self, *args, **kwargs):
         """Execute task."""
         with self.app.flask_app.test_request_context():
-            self.app.flask_app.try_trigger_before_first_request_functions()
```

Comment:
Indeed, it looks like that it was removed here: https://github.com/pallets/flask/pull/4621

And it looks unused...